modules/samples/artifacts/PublishHttpWithExpiredToken/PublishHttpWithExpiredToken.siddhi

```diff
@@ -0,0 +1,93 @@
+@App:name("PublishHttpWithExpiredToken")
```

Comment:
Why is this sample called `PublishHttpWithExpiredToken`? Where do we have an expired token?